SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1273,7 +1273,12 @@ public async Task<IActionResult> HeatingPattern_Post(HeatingPatternViewModel vie
             }
             
             return await UpdatePropertyAndRedirect(
-                p => p.HeatingPattern = viewModel.HeatingPattern,
+                p =>
+                {
+                    p.HeatingPattern = viewModel.HeatingPattern;
+                    p.HoursOfHeatingMorning = viewModel.HoursOfHeatingMorning;
```

Comment:
What if the user chooses `Other`, enters some numbers, and then chooses `All day and all night`? I think we'll save the heating hours in that case when we shouldn't

Reply:
In `UpdatePropertyAndRedirect`, right after `update` is called (i.e. these assignments), `ResetAllUnusedFields` is called which would delete the fields we do not want anyway.